### DISCUSSION TITLE
Welcome to the Babelfish Discussions!

### DISCUSSION THREAD

**Discussion by suprio-amzn** [babelfish-for-postgresql/babelfish-for-postgresql#10] [Announcements]
👋 Welcome!

We will begin using Discussions for Babelfish in this GitHub project as a place to connect with other collaborators. You can use discussions to:

* Ask questions about Babelfish you’re wondering about
* Share new project ideas to enhance Babelfish to become more useful for others
* Engage with other members of our collaboration group
* Be open-minded collaborators to build a stronger product together 💪

The project will use the Discussions forum to:

* Publish announcements on decisions affecting the Babelfish community's current projects
* Publish items related to future roadmap
* Initiate technical design discussions that we want you (the community members) to be involved at all levels - big or small to make the design and final product better

To get started, please comment below with an introduction of yourself and tell us about what you do plan to do in this community collaboration group.

🔗 If you have been using issues in the [public project](https://github.com/babelfish-for-postgresql/babelfish_extensions/issues) which you think could benefit from a detailed discussion, move such questions and community conversations to the Discussions area here. Tracking important topics that need more involved interaction is the intended use of this Discussions forum.

**Comment by vigneshkumar2016** [comment]
Do babelfish support the light weight debian flavours. as of now could see its well supported with ubuntu. when trying on debian light weight os. getting parsing error on ANTLR. Included the PGXS and then trying to CMake. even then the build is failing. any suggestions how to move here.

**Reply by suprio-amzn** [reply]
@vigneshkumar2016: This is a discussion forum and the above post is an announcement.

Please pose your queries of the above type in https://github.com/babelfish-for-postgresql/babelfish_extensions/issues.

**Comment by staticlibs** [comment]
@vigneshkumar2016 

Can you post the ANTLR error you are getting? In general Babelfish is buildable on any modern linux, but changes to compiler flags in Makefiles are almost always required. With ANTLR you need to either use `4.9.3` C++ runtime lib (with in-tree generator JAR) or apply minor patches to use newer ANTLR.
